First Draft for Chapter Site (#1)
OWASP/www-chapter-stuttgart: index.md

---
layout: col-sidebar
title: OWASP Stuttgart
tags: germany stuttgart
region: Europe
country: Germany
meetup-group:
---
## Welcome

Welcome to _OWASP Stuttgart Chapter_, a regional city chapter within [OWASP Germany](https://owasp.org/www-chapter-germany/). Our Chapter serves southern Germany particular within the Swabia region as a platform to discuss and share topics all around information and application security.

> Anyone with an interested and enthusiastic about application security is welcome. All meetings are free and open. **You do not have to be an OWASP member**.

Reviewer: ```suggestion
Anyone interested and enthusiastic about application security is welcome. All meetings are free and open. **You do not have to be an OWASP member**.
```

Referrals to this website or to individual meetings to colleagues or acquaintances are welcome.

## Upcoming Events

Check out our [Upcoming Events](https://www.meetup.com/de-DE/OWASP-Stuttgart/events/)

## Information about Stuttgart Chapter Events

### When?

We will be running quarterly chapter meetups with the option to have additional events such as hands-on workshops. 

> The dates will be picked randomly to avoid recurring conflicts with other events (meetups, private obligations, holiday etc.). Typically, we chose a day from Tuesday to Thursday in the week, since our experience in organizing events is, that Mondays and Fridays have a very high no-show rate 😬

Reviewer: ```suggestion
The dates will be picked randomly to avoid recurring conflicts with other events (meetups, private obligations, holiday etc.). Typically, we choose a day from Tuesday to Thursday for our events, as our experience in organizing them has shown that Mondays and Fridays often have a very high no-show rate. 😬
```

> Typically, the event starts at 18:00 with arriving and socialising. Starting at 18:30 with the talk of our guest speaker or whatever we've planned for the particular event.

Reviewer: Events usually start at 18:00 with guests arriving and socialising. By 18:30, we proceed with the talk from our guest speaker or dive into the planned activities for the evening.

> Typically, the event starts at 18:00 with arriving and socialising. Starting at 18:30 with the talk of our guest speaker or whatever we've planned for the particular event.

Reviewer: ```suggestion
Events usually start at 18:00 with guests arriving and socialising. By 18:30, we proceed with the talk from our guest speaker or dive into the planned activities for the evening.
```
